data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/utils/DocumentDBSourceAggregateMetrics.java

```diff
@@ -17,8 +17,7 @@ public class DocumentDBSourceAggregateMetrics {
     private static final String DOCUMENT_DB_EXPORT_5XX_ERRORS = "export5xxErrors";
     private static final String DOCUMENT_DB_EXPORT_4XX_ERRORS = "export4xxErrors";
     private static final String DOCUMENT_DB_EXPORT_API_INVOCATIONS = "exportApiInvocations";
-
-
+    private static final String DOCUMENT_DB_EXPORT_PARTITION_QUERY_COUNT = "exportPartitionQueryCount";
```

Comment:
What is this aggregated metric used for?

Reply:
This is to track number of export partition query created. When the collection size if very large (10's to 100's of GB), the scan of collection takes in minutes to hours to create query partition. We want to provide visibility to customer of query partition getting created and pipeline is not stuck.